### PR TITLE
Migrate the README to markdown

### PR DESCRIPTION
**Pull request by childish-sambino**
PyPI guide: https://packaging.python.org/guides/making-a-pypi-friendly-readme/
Sphinx Markdown support guide: http://www.sphinx-doc.org/en/master/usage/markdown.html

**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Developer Preview status. Such products are explicitly documented as
+Beta/Developer Preview both in the Twilio docs and console, as well as through
+in-line code documentation here in the library.
+
+## Installation
+
+Install from PyPi using [pip](http://www.pip-installer.org/en/latest/), a
+package manager for Python.
+
+    pip install twilio
```
Comment (thinkingserious) [inline README.md]: Any reason to not format these like: 
`pip install twilio` or 
```bash
pip install twilio
```

Comment (childish-sambino) [reply README.md]: No reason. I just restored the old version of this file (and added a couple things that were in the _rst_ variant): https://github.com/twilio/twilio-python/blob/320e3913cce3cc50694d28378d598b0444abe802/README.md

Comment (thinkingserious) [reply README.md]: I would say to be consistent (e.g. [PHP](https://github.com/twilio/twilio-php)) we should use the pretty code formatting stuff.

Comment (thinkingserious) [reply README.md]: Hmm, but in the preview it still looks good. (shrug)

**Comment by childish-sambino** [comment]
Don't merge yet. Some issue with sphinx doc generation referencing the _rst_ README.

**Pull request merged**